0000-remove-unused-keys-for-makers-in-bad-standing.md

```diff
@@ -0,0 +1,78 @@
+# HIP XXXX: Wipe unused onboarding keys for makers in bad standing
```

Comment:
probably needs a more clear title. most of the community doesn't know what an onboarding key is (unfortunately). 

How about something like:

```
Remove Onboarding Capability for Hotspots Manufactured by Makers in Bad Standing
```